backend/src/modules/organization/repositories/organization-repository.service.ts

```diff
@@ -12,20 +16,56 @@ export class OrganizationRepositoryService implements IOrganizationRepository {
     private readonly organizationRepository: Repository<OrganizationEntity>,
   ) {}
 
-  // TODO: Database related operations
-  create(organization: IOrganizationModel): Promise<IOrganizationModel> {
-    throw new Error('Method not implemented.');
+  public async create(
+    organization: Omit<IOrganizationModel, 'id'>,
+  ): Promise<IOrganizationModel> {
+    // create organization entity
+    const newOrganizationEntity = toOrganizationEntity(organization);
+
+    // save organization entity
+    const organizationEntity = await this.organizationRepository.save(
+      newOrganizationEntity,
+    );
+
+    // return organization model
+    return toOrganizationModel(organizationEntity);
   }
 
-  update(description: string): Promise<IOrganizationModel> {
-    throw new Error('Method not implemented.');
+  public async update(
+    id: string,
+    description: string,
+  ): Promise<IOrganizationModel> {
+    // update organization entity
+    await this.organizationRepository.update({ id }, { description });
```

Comment:
Probabil o sa punem un BaseDAO abstact in care sa avem logica gen asta de `update`, sau `getManyPaginated`?